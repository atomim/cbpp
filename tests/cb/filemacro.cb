AddText "this is file is "+__FILE__
