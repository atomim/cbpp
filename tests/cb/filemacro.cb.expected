AddText "this is file is "+"filemacro.cb"
